zerver/context_processors.py

```diff
@@ -56,19 +56,19 @@ def zulip_default_context(request):
     """
     realm = get_realm_from_request(request)
 
+    realm_uri = settings.SERVER_URI
+    realm_name = None
+    realm_icon = None
+    realm_description = None
+    realm_invite_required = False
```

Comment:
Hmm. This structure feels less nice to read than the existing code. I guess the alternative is to add type annotations on the things that are optional -- like
```
  realm_name = realm.name  # type: Optional[str]
  realm_icon = get_realm_icon_url(realm)  # type: Optional[str]
```
which isn't terrible, though it gets a little annoying on `realm_description`. Not sure I have a preference between these, though.